index.html

```diff
@@ -1,28 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    <header class="menu-wrap">
-        <nav class="nav-bar">
-            <a href="#" class="brand">My Logo</a>
-            <div class="hamburger">
-              <span class="bar"></span>
-              <span class="bar"></span>
-              <span class="bar"></span>
-            </div>
-        </nav>
-      </header>
-   <section class="wrapper">
-     <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
-     <p class="intro">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p><br>
-      <a href="#" class="connect">LET'S CONNECT</a><br>
-      <div class="socials">
+  <header class="menu-wrap">
+    <nav class="nav-bar">
+      <a href="#" class="brand">My Logo</a>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div>
+        <ul class="menu">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="wrapper">
+    <h1 class="title">I'm Jesse Glad to see you!</h1>
+    <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+    </p><br>
+    <a href="#" class="connect">LET'S CONNECT</a><br>
```

Comment:
- [ OPTIONAL]KIndly try to match your design to the figma one , I can see that your 'LET'S CONNECT' button is not looking the same as the one the Figma design when we move on mobile version  :wink: 
Your Design
![Screenshot from 2023-02-25 19-46-19](https://user-images.githubusercontent.com/88894525/221371953-e878771e-5aaf-4a56-842c-20c8f53b8798.png)

Figma Design
![Screenshot from 2023-02-25 19-47-06](https://user-images.githubusercontent.com/88894525/221371979-b1e99d9c-9965-4c96-b141-a82d56bbeccd.png)

_Note: I can see that even your `h2` (About Myself) also is not the same as on Figma design_